include/config/config_game.h

```diff
@@ -72,22 +72,22 @@
 // #define DIALOG_INDICATOR
 
 /**
- * Include the English characters that were missing from US segment2
- * J, Q, V, X, Z, ¨, !, !!, ?, &, %, ., and the beta key.
- * [MAKE SURE TO INCLUDE EU AND JP/SH BASEROMS IN THE REPO TO OBTAIN THE ASSETS]
- * If this is disabled, backup assets will be used.
+ * Adds support for multiple languages to the game.
  */
-// #define COMPLETE_EN_US_SEGMENT2
+#define MULTILANG
 
 /**
- * Removes multi-language cake screen.
+ * Toggles for individual languages. Only works when MULTILANG is enabled.
  */
-#define EU_CUSTOM_CAKE_FIX
+#define ENABLE_FRENCH
+#define ENABLE_GERMAN
+#define ENABLE_JAPANESE
```

Comment:
japanese shouldn't be enabled by default, since it requires a japanese baserom

also, it would be neat if, when this is enabled and there is no jp baserom, there was an assert letting the hacker know about this, rather than it just failing because of a missing texture. might be outside the scope of this PR though